### PR TITLE
add mapping for C,C1,C+E and C1+E

### PR DESCRIPTION
**Pull request by rossfauldsbjss**
MES-2707

Added mappings + tests for category C and it's subcategories (C1,C+E, C1+E).
Upon analysis it was seen that C and C1 are identical, and C+E and C1+E have identical so I chose not to duplicate helper classes for C1 and C1+E but instead re-use C and C+E.

If the sub categories diverge in future we would need to create new helpers.

## Pull Request checklist

- [X] [WIP] tag removed from PR title
- [X] PR has an understandable description

## Git feature branch checklist

- [X] branch name comply with our branching strategy
- [X] git branch contains relevant JIRA ticket number
- [X] branch rebased against the latest develop

## Sign off process checklist

- [X] Code has been tested manually
- [X] PR link added to JIRA

### CODE REVIEW

**Review by faggc — APPROVED** [review]

File: src/functions/uploadToRSIS/application/data-mapper/__tests__/cat-c-mapper.spec.ts [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+      { col: 'JUDGEMENT_CROSS_SERIOUS', val: 0 },
+      { col: 'POSITIONING_NORMAL_SERIOUS', val: 0 },
+      { col: 'POSITIONING_LANE_SERIOUS', val: 0 },
+      { col: 'PEDESTRIAN_CROSSING_SERIOUS', val: 0 },
+      { col: 'POSTITION_STOPS_SERIOUS', val: 0 },
+      { col: 'AWARENESS_PLAN_SERIOUS', val: 0 },
+      { col: 'ANCILLARY_CONTROLS_SERIOUS', val: 0 },
+      { col: 'REV_LEFT_TRAIL_CONT_DANGEROUS', val: 0 },
+      { col: 'REV_LEFT_TRAIL_OBSER_DANGEROUS', val: 0 },
+    //   { col: 'CONTROL_STOP_PROMPT_DANGEROUS', val: 0 },
```
Comment (TomBillingtonUK) [inline src/functions/uploadToRSIS/application/data-mapper/__tests__/cat-c-mapper.spec.ts]: Can we remove these or are they needed?

File: src/functions/uploadToRSIS/application/data-mapper/__tests__/cat-c1-mapper.spec.ts [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+      { col: 'JUDGEMENT_CROSS_SERIOUS', val: 0 },
+      { col: 'POSITIONING_NORMAL_SERIOUS', val: 0 },
+      { col: 'POSITIONING_LANE_SERIOUS', val: 0 },
+      { col: 'PEDESTRIAN_CROSSING_SERIOUS', val: 0 },
+      { col: 'POSTITION_STOPS_SERIOUS', val: 0 },
+      { col: 'AWARENESS_PLAN_SERIOUS', val: 0 },
+      { col: 'ANCILLARY_CONTROLS_SERIOUS', val: 0 },
+      { col: 'REV_LEFT_TRAIL_CONT_DANGEROUS', val: 0 },
+      { col: 'REV_LEFT_TRAIL_OBSER_DANGEROUS', val: 0 },
+    //   { col: 'CONTROL_STOP_PROMPT_DANGEROUS', val: 0 },
```
Comment (TomBillingtonUK) [inline src/functions/uploadToRSIS/application/data-mapper/__tests__/cat-c1-mapper.spec.ts]: same here ?

**Review by TomBillingtonUK — APPROVED** [review]
Looks good - There are a few  lines of commented out code in the mapper files. Have commented on the first 2 , not sure if they can or should be removed

File: src/functions/uploadToRSIS/application/data-mapper/__tests__/cat-c-mapper.spec.ts (hunk 1 above)
Comment (rossfauldsbjss) [reply src/functions/uploadToRSIS/application/data-mapper/__tests__/cat-c-mapper.spec.ts]: Copied from existing code but i'll clean them all out.

File: src/functions/uploadToRSIS/application/data-mapper/__tests__/cat-c1-mapper.spec.ts (hunk 2 above)
Comment (rossfauldsbjss) [reply src/functions/uploadToRSIS/application/data-mapper/__tests__/cat-c1-mapper.spec.ts]: same response. will remove

**Pull request merged**